src/main/webapp/index.html

```diff
@@ -43,6 +44,9 @@ <h4 class="text-center lead">InterviewMe is a platform for conducting mock techn
           practice their skills. We aim to connect users with live video chats in order to encapsulate the true feeling
           and pace of a technical interview.
         </h4>
+        <div id="loggedOut">
```

Comment:
Ok l get your point. The name is just kind of ambiguous. Change to login-message-container or something that specifies what it does. Also in HTML we should try to stick with the dash method for variable names.

Reply:
Done